.changeset/README.md

```diff
@@ -1,8 +1,32 @@
-# Changesets
+# Changeset sjabloon
 
-Hello and welcome! This folder has been automatically generated by `@changesets/cli`, a build tool that works
-with multi-package repos, or single-package repos to help you version and publish your code. You can
-find the full documentation for it [in our repository](https://github.com/changesets/changesets)
+Kopieer en plak het onderstaande sjabloon. Je kunt hiervoor de kopieer knop linksboven in het template gebruiken.
 
-We have a quick list of common questions to get you started engaging with this project in
-[our documentation](https://github.com/changesets/changesets/blob/main/docs/common-questions.md)
+```markdown
+---
+"@organisatie/package-naam": major
+---
+
+- 
```

Comment:
Ik zou in plaats van het streepje gewoon een woord zetten, zoals `Beschrijving`.